.ci/unit-tests/ansible.yml

```diff
@@ -0,0 +1,12 @@
+platform: linux
+inputs:
+  - name: magic-modules
+image_resource:
+  type: docker-image
+  source:
+    repository: python
+    tag: '2-jessie'
```

Comment:
I'd use 2.7-alpine - you only get to specify the stuff that's in your tag name and I assume ansible would be angry if suddenly python was 2.6.  Also, I like alpine over jessie, because alpine is the tiny tiny tiny linux and jessie is a big ol' debian, and if you don't need any of that debian stuff then alpine is a winner.

Reply:
As discussed, we're going with Jessie because Alpine is too small.